### ISSUE TITLE
Accuracy is much lower in my case

### ISSUE THREAD

**Issue by giuliaviolatto** [jrieke/shape-detection#3]
[Output_performance_simple_model.txt](https://github.com/jrieke/shape-detection/files/4633517/Output_performance_simple_model.txt)
Hi,

I tried your notebook using tensorflow.keras instead of Keras. (Tensorflow-gpu v 2.1.0). 
I only modified the code for the model in Cell [81], where I removed the parameter **dim_ordering='tf'** because it is not needed (and available) in tensorflow.keras. 
I tried to train the model for 20 epochs multiple times and I get very different results with respect to your notebook, in terms of "Accuracy shapes" and "Accuracy colors". (see Output_performance_simple_model.txt ). Do you know what can be the problem? 
Can it be only the different version of Keras that I used?

Best regards,
Giulia

**Comment by jrieke** [comment]
Sorry, missed this issue. Yes, something must have changed. I added a note to the readme a while ago:

Note: Some people reached out and told me that they can't replicate some of the results in this repo. I couldn't find the reason, potentially one of the dependencies changed since I did this project a few years ago. Unfortunately I don't have any notes on which exact setup I used back then. If you find anything, please reach out to me :)
